src/components/modules/Navbar/NavItem.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+import { HTMLAttributeAnchorTarget, ReactNode } from 'react'
+
+type Props = {
+  children: ReactNode
+  href: string
+  className?: string
+  target?: HTMLAttributeAnchorTarget
+}
+
+export const NavItem = ({ children, href, className, target }: Props) => {
+  return (
+    <li className={`${className} mt-8 lg:ml-10 lg:mt-0 xl:ml-16 xl:mt-0`}>
+      <Link href={href}>
+        <a target={target}>{children}</a>
```

Comment:
why there is an `a` tag inside a `Link`?

Reply:
dari sini
https://github.com/nodefluxio/weber/blob/main/frontend/app/components/modules/Navbar/NavItem.tsx

Reply:
wkwkw dosa itumah jgn diikutin, gimana klo manfaatin component UnstyledLink?